pkg/network/ovs_test.go

```diff
@@ -39,10 +39,9 @@ var _ = Describe("Testing Ovs", func() {
 		Context("when there is previous value, but the new one is empty (removing component)", func() {
 			prev := &opv1alpha1.NetworkAddonsConfigSpec{Ovs: &opv1alpha1.Ovs{}}
 			new := &opv1alpha1.NetworkAddonsConfigSpec{}
-			It("should fail", func() {
+			It("should success", func() {
 				errorList := changeSafeOvs(prev, new)
-				Expect(len(errorList)).To(Equal(1), "validation of safe change failed due to an unexpected error: %v", errorList)
-				Expect(errorList[0].Error()).To(Equal("cannot modify Ovs configuration once it is deployed"))
+				Expect(len(errorList)).To(Equal(0), "validation of safe change failed due to an unexpected error: %v", errorList)
```

Comment:
No need to check for len, just check that the list is empty: https://github.com/kubevirt/cluster-network-addons-operator/pull/287/files#diff-2297106a77e4f1babcc0c6b321b7954dR35